index.html

```diff
@@ -40,7 +84,7 @@
                 <div class="col-lg-6 hero-content" data-parallax="0.2">
                     <h1>Ready to kick-start your next reading binge?</h1>
                     <p class="tagline">
-                        Lightning fast with a slick design, Kavita is a rocket fueled self-hosted digital library which supports a vast array of file formats. Install to start reading and share your server with your friends.
+                        Lightning fast and privacy-friendly, Kavita is a self‑hosted digital library for EPUB, PDF, comics and manga — with built‑in readers (webtoon mode), OPDS, full‑text search and rich metadata. Install and share your server in minutes.
```

Comment:
Webtoon mode seems weird here

Reply:
FTS also might be misleading, users might think we search within books

Reply:
Webtoon is the word, but it looks out of place here. Might be better to word as: "...with built‑in readers (single, double page, and webtoon mode)..."

This way we showcase double mode as well and also it's not out of place.